docs/sections/user_guide/cli/drivers/cdeps/help.rst

```diff
@@ -0,0 +1 @@
+../shared/help.rst
```

Comment:
In each of the `cli/drivers/*` directories, symlink to `help.rst` and, below, `schema-options.rst`. These unfortunately cannot be referred to with relative links like `../shared/idempotent.rst` (see the `index.rst` diff just below) because they themselves contain links to file to include (`help.cmd`, `help.out`, etc.), and Sphinx looks for those files in the directory containing the including file, rather than in the directory containing `index.rst`. Since `drivers/shared/` does not contain `help.cmd` et al., the doc build fails. By symlinking `help.rst` into e.g. the `cdeps/` directory, `help.rst` can find the CDEPS-specific `help.cmd`, `help.out`, etc. files and include them properly. The upshot is that the `drivers/*` directories all look more similar to one another, with duplicate content factored out and fewer differences in content that could lead to copy/paste errors.

When creating a new driver subdirectory, we'll have to keep in mind that several of its files will be symlinks.